contracts/Operator.sol

```diff
@@ -24,9 +24,9 @@ contract Operator is Upgradeable {
   mapping(uint256 => Proposal) public proposals;
   uint256 public proposalsCount;
 
-  event ProposalCreated(uint256 id, address creator, uint recipient, uint256 amount);
-  event ProposalVoted(uint256 id, address voter, uint256 totalVotes);
-  event ProposalExecuted(uint256 id, uint recipient, uint256 amount);
+  event ProposalCreated(uint256 id, address creatorAccount, uint256 recipientId, uint256 amount);
```

Comment:
What about `recipientId` -> `contributorId`?

Reply:
In these events we have two different "contributors" 
one is the recipient and one is the voter - that's why I'd use voterId and recipientId. 
?